style.css

```diff
@@ -353,3 +353,114 @@ hr {
   display: flex;
   align-items: flex-end;
 }
+
+.covid {
+  background-color:#5BC6F9;
+  position: relative;
+}
+
+.covid h1 {
+  color:#fff;
+  text-align: start;
+  padding-top: 2em;
+  padding-left: 6.25em;
```

Comment:
Have you considered to instead of using `padding-left` and `padding-right` to then flip them in a `html[dir="rtl"]` to use something like [`padding-inline-start`](https://developer.mozilla.org/en-US/docs/Web/CSS/padding-inline-start) and [`padding-inline-end`](https://developer.mozilla.org/en-US/docs/Web/CSS/padding-inline-end)?